crates/telio-sockets/src/protector/linux.rs

```diff
@@ -44,7 +44,6 @@ impl Protector for NativeProtector {
 }
 
 fn set_fwmark(fd: i32, fwmark: u32) -> io::Result<()> {
-    let fwmark = fwmark;
```

Comment:
Was there any specific reason why fwmark was added redundantly here ? Or just something that was missed during reviews ?

Reply:
I can't figure out any reason why this was done, since this isn't a pointer or anything. The function call already creates a new instance of that value on the stack, so there shouldn't be any way that this provides some special functionality. Clippy threw an error here, so that's why I removed it in the first place. I assume that even if this had some special side effect, there should have been at least a comment here.